modules/op-resolver-http.adoc

```diff
@@ -0,0 +1,94 @@
+// This module is included in the following assemblies:
+// * create/remote-pipelines-tasks-resolvers.adoc
+
+// // *openshift_pipelines/remote-pipelines-tasks-resolvers.adoc
+:_mod-docs-content-type: PROCEDURE
+[id="resolver-http-specify_{context}"]
+= Specifying a remote pipeline or task with the HTTP Resolver
+
+When creating a pipeline run or task run, you can specify a remote pipeline or task from an HTTP or HTTPS URL.
+
+.Prerequisites
+
+* You can optionally configure the service account for the HTTP resolver.
+
+.Procedure
+
+* To specify a remote pipeline or task from an HTTP URL, use the following reference format in the `pipelineRef` or `taskRef` spec:
```

Comment:
```suggestion
* To specify a remote pipeline or task from an HTTP URL, use the following format in the `pipelineRef` or `taskRef` spec:
```

Reply:
In my opinion, the `Ref` piece is taken care of in the spec, and this sample is just a format in a procedure so calling it a "reference format" seems unnecessary, but take it or leave it :)